sdk/python/kfp/dsl/_metrics_utils.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kfp.dsl import _artifact_utils
+from typing import Any, List
+
+class ConfidenceMetrics(object):
```

Comment:
The fact that `ConfidenceMetrics` and `ConfusionMatrix` shares some common methods makes me think that they should inherit from a common base class. WDYT?

Reply:
Sounds good, updated.